### PR TITLE
chore: add libs

### PR DESCRIPTION
**Pull request by IKatsuba**

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -139,32 +139,6 @@
         }
       }
     },
-    "ssr-article-e2e": {
```
Comment (MarsiBarsi) [inline angular.json:142]: надо нам `nx g application name --e2eTestRunner=none --routing=true --style=less --skipTests=true` прибить куда-нибудь

Comment (IKatsuba) [reply angular.json:142]: там есть варик написать схематики (nx же). Мб потом

**Review by MarsiBarsi — APPROVED** [review]

**Comment by IKatsuba** [comment]
:tada: This pull request is included in version webpack-plugin@0.1.0-alpha.5 :tada:

The release is available on [GitHub release](<github_release_url>)

**Comment by IKatsuba** [comment]
:tada: This pull request is included in version core@0.1.0-alpha.4 :tada:

The release is available on [GitHub release](<github_release_url>)

**Comment by IKatsuba** [comment]
:tada: This pull request is included in version angular@0.1.0-alpha.7 :tada:

The release is available on [GitHub release](<github_release_url>)

**Pull request merged**